ansibullbot/triagers/plugins/label_commands.py

```diff
@@ -7,24 +7,43 @@ def get_label_command_facts(issuewrapper, meta, module_indexer, core_team=[], va
     add_labels = []
     del_labels = []
 
-    wl = [
-        'needs_triage',
-        'test',
-        'module',
-        'cloud',
+    namespace_labels = [
+        'aci',
+        'avi',
         'aws',
         'azure',
+        'cloud',
+        'cloudstack',
         'digital_ocean',
         'docker',
+        'f5',
         'gce',
+        'infoblox',
+        'jboss',
+        'meraki',
+        'netapp',
+        'networking',
+        'nxos',
         'openstack',
+        'ovirt',
+        'ucs',
+        'unarchive',
```

Comment:
why is unarchive a namespace label?

Reply:
It's a "module community" label ? Much like the `xml` label.
It allows to track issues/PRs related to specific modules.

Reply:
I don't mind removing them from the list if that is an issue.

IMO a blacklist protecting special labels is more convenient than listing namespace labels (and others).

Reply:
Idea: maybe we could introduce m:<module> for module namespaces and only have ansibot add those when they were pre-created ? This way it's still on-demand, but integrates automatically with ansibot.

Reply:
works for me, shipit.